exec/vector/src/main/java/org/apache/drill/exec/vector/accessor/TupleAccessor.java

```diff
@@ -48,9 +48,21 @@
 
     MaterializedField column(int index);
 
-    MaterializedField column(String name);
+    /**
+     * Returns {@code MaterializedField} instance from schema using the name path specified in param.
+     *
+     * @param name full name path of the column in the schema
+     * @return {@code MaterializedField} instance
+     */
+    MaterializedField column(String[] name);
```

Comment:
This is not right. As explained earlier, a tuple is a specialized map that allows both name and indexed access, but only to direct children. The name above should never be compound. Instead, the code must work the hierarchy itself (or we'd need to add a method to do so.)

Note that this has been revised/replaced in PR #914.